src/components/OrgPostCard/OrgPostCard.tsx

```diff
@@ -134,43 +134,55 @@ function OrgPostCard(props: OrgPostCardProps): JSX.Element {
               </a>
             </div>
           </div>
-          <p>
-            {t('author')}:<span> {props.postAuthor}</span>
-          </p>
-
-          {togglePost === 'Read more' ? (
-            <p data-testid="toggleContent">
-              {props.postInfo.length > 43
-                ? props.postInfo.substring(0, 40) + '...'
-                : props.postInfo}
-            </p>
-          ) : (
-            <p data-testid="toggleContent">{props.postInfo}</p>
-          )}
-          <button
-            role="toggleBtn"
-            className={`${
-              props.postInfo.length > 43
-                ? styles.toggleClickBtn
-                : styles.toggleClickBtnNone
-            }`}
-            onClick={handletoggleClick}
-          >
-            {togglePost}
-          </button>
           {/* {props.postInfo.length > 43 && (
             <button role='toggleBtn' className={styles.toggleClickBtn} onClick={handletoggleClick}>
               {togglePost}
             </button>
           )} */}
           {/* <p>{props.postInfo}</p> */}
+          {props.postPhoto ? (
+            <p>
+              <span>
+                {' '}
+                <img
+                  className={styles.postimage}
+                  alt="image not found"
+                  src={props?.postPhoto}
+                />
+              </span>
+            </p>
+          ) : (
+            <img
+              src="https://via.placeholder.com/200x100"
```

Comment:
We need to be using place holder images that are a part of our code base. Relying on external sources that could disappear is not a good practice.

Reply:
Yes Sir 100%, I had the same doubt when using this link in the code base. but in many components this place holder images are being used. 
maybe sir, Instead of getting this Image from a link I can download this image in assets folder and use the downloaded one in the code base. what do you think ? @palisadoes 

Example of place holder Images being used in different files : 
![doubt](https://user-images.githubusercontent.com/115649011/227738490-b8d321d4-91ca-4ff1-a244-23657977c71b.png)